src/m365/spo/commands/list/list-roleassignment-add.ts

```diff
@@ -170,7 +170,7 @@ class SpoListRoleAssignmentAddCommand extends SpoCommand {
       }
       else if (args.options.groupName) {
         args.options.principalId = await this.GetGroupPrincipalId(args.options);
-        this.AddRoleAssignment(requestUrl, logger, args.options);
+        await this.AddRoleAssignment(requestUrl, logger, args.options);
```

Comment:
@milanholemans I think this is a bug that we overlooked while refactoring to async 🤔. 
I noticed it while testing the commands after refactoring this one.

Reply:
Hmm strange, good catch @Adam-it!